Plugins/Public/event/Main.cpp

```diff
@@ -914,7 +914,10 @@ void __stdcall SPMunitionCollision(struct SSPMunitionCollisionInfo const & ci, u
 		}
 		if (iLootCount == 0)
 		{
+			PrintUserCmdText(iClientID, L"%s's cargo is now full.", reinterpret_cast<const wchar_t*>(Players.GetActiveCharacterName(iSendToClientID)));
```

Comment:
we don't want to run this or the NNMessage if iClientID == iSendToClientID (same in minecontrol). I suggest you make a if (iClientID != iSendToClientID) block and move the stuff involving iSendToClientID into that.